src/MensaWidget/MWDataGettingWrapperTest.class.st

```diff
@@ -0,0 +1,49 @@
+"
+A MWDataGettingWrapperTest is a Test Case for the MWDataGettingWrapper.
+
+Instance Variables
+	dataGettingWrapper:		<MWDataGettingWrapper>
+
+dataGettingWrapper
+	- a DataGettingWrapper instance on which the tests will be performed
+
+"
+Class {
+	#name : #MWDataGettingWrapperTest,
+	#superclass : #TestCase,
+	#instVars : [
+		'dataGettingWrapper'
+	],
+	#category : #'MensaWidget-Tests',
+	#'squeak_changestamp' : 'Louis Christ 5/27/2024 19:35'
+}
+
+{
+	#category : #'as yet unclassified',
+	#'squeak_changestamp' : 'Louis Christ 5/27/2024 19:39'
+}
+MWDataGettingWrapperTest >> testCollectionClasses [
+	
+	dataGettingWrapper todaysMeals do: [ :each |
+		self assert: each class = MWMeal ]
+]
+
+{
+	#category : #'as yet unclassified',
+	#'squeak_changestamp' : 'Louis Christ 5/27/2024 19:42'
+}
+MWDataGettingWrapperTest >> testCollectionValues [
+	
+	dataGettingWrapper todaysMeals do: [ :each |
+		each instanceVariableValues do: [ :value | 
+			self assert: value = nil not]]
+]
```

Comment:
I do not think that this is an appropriate test case, since there might be special objects or cases that do not require one of the instance variables or where the data is simply not provided by the API.

Reply:
So should we delete it completly or rather replace it with something?

Reply:
To be honest, since the functionality is so limited without the actual Parser and APIWrapper, I'd say we do extended tests after having them all together... so I would remove this test

Reply:
done